### PR TITLE
Added safer rust-based webhook

### PR DESCRIPTION
**Pull request by terhechte**
Not deployed yet. Already compiled for x64 Linux, binary is part of the PR.

### CODE REVIEW

**Comment by terhechte** [comment]
Only 12 lines more code than the Python Version, not bad I'd say. And much Safer :)

File: rust_webhook/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+[package]
+name = "rust_webhook"
+version = "0.1.0"
+authors = ["Benedikt Terhechte <terhechte@gmail.com>"]
+edition = "2018"
```
Comment (BasThomas) [inline rust_webhook/Cargo.toml]: 🤔

Comment (terhechte) [reply rust_webhook/Cargo.toml]: That's auto-generated by Rust.

**Comment by terhechte** [comment]
> Safer perhaps, but not more understandable for me 🤪

Well, we can also write another one in SwiftNIO or Vapor (where "we" means "you" ;) As long as you generate a compiled binary for x64 Linux, I'm happy to run it ;) )

File: rust_webhook/Cargo.toml (hunk 1 above)
Comment (BasThomas) [reply rust_webhook/Cargo.toml]: What's with the "edition"?

**Comment by BasThomas** [comment]
> Well, we can also write another one in SwiftNIO or Vapor (where "we" means "you" ;) As long as you generate a compiled binary for x64 Linux, I'm happy to run it ;) )

Putting me on the spot here 😅

File: rust_webhook/Cargo.toml (hunk 1 above)
Comment (terhechte) [reply rust_webhook/Cargo.toml]: "Rust ships releases on a six-week cycle. This means that users get a constant stream of new features. This is much faster than updates for other languages, but this also means that each update is smaller. [...] Every two or three years, we'll be producing a new edition of Rust. Each edition brings together the features that have landed into a clear package, with fully updated documentation and tooling. New editions ship through the usual release process. [...] When a new edition becomes available in the compiler, crates must explicitly opt in to it to take full advantage. This opt in enables editions to contain incompatible changes, like adding a new keyword that might conflict with identifiers in code, or turning warnings into errors."

So every 6 weeks a new release with new features, but breaking changes only every 2-3 years, and even then you can just instruct the compiler to use an older edition so you're not forced into upgrading our project. Ever.

Comment (terhechte) [reply rust_webhook/Cargo.toml]: https://doc.rust-lang.org/edition-guide/editions/index.html

**Pull request merged**